tcms/testruns/models.py

```diff
@@ -256,9 +257,15 @@ class TestExecution(TCMSActionModel):
         on_delete=models.CASCADE,
     )
     case_text_version = models.IntegerField()
-    close_date = models.DateTimeField(null=True, blank=True)
+    start_date = models.DateTimeField(null=True, blank=True, db_index=True)
+    stop_date = models.DateTimeField(null=True, blank=True, db_index=True)
     sortkey = models.IntegerField(null=True, blank=True)
 
+    @property
+    def _actual_duration(self):
+        duration_difference = self.start_date - self.stop_date
```

Comment:
You don't need an intermediate variable to store the results, `return` the expression directly.